model/riscv_insts_vext_utils.sail

```diff
@@ -0,0 +1,703 @@
+/* ************************************************************************** */
+/* This file implements functions used by vector instructions.                */
+
+/* ************************************************************************** */
+
+
+/* Vector mask mapping */
+mapping maybe_vmask : string <-> bits(1) = {
+  ""              <-> 0b1, /* unmasked by default */
+  sep() ^ "v0.t"  <-> 0b0
+}
+
+/* Check for valid vsew and lmul values */
+val vcheck_vsew_lmul : (int, real) -> Retired
```

Comment:
Retired feels wrong for these, they should be bool? Instructions retire, helper functions return booleans/integers/optionals.